web/src/beta/features/Editor/index.tsx

```diff
@@ -75,6 +74,7 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
     isVisualizerReady,
     visualizerRef,
   });
+  const { scene, selectedSceneSetting, handleSceneSettingSelect } = useScene({ sceneId });
```

Comment:
You've set up this useScene, but then only pass the scene to the left panel, and then still use the right panel's direct use of the scene api to fetch the scene. 
If you want to consolidate scene in this useScene, please at least update the files you are working in in this PR to match.

Aka, if useLeftPanel will take a scene prop and not access the scene api directly, make sure the useRightPanel is updated accordingly since you are already modifying both in this PR.

Reply:
Done